.gitlab-ci.yml

```diff
@@ -93,7 +89,6 @@ build-aarch64-darwin:
         --keep CI_PROJECT_DIR \
         --keep MACOSX_DEPLOYMENT_TARGET \
         --keep GHC_VERSION \
-        --keep CABAL_PROJECT \
```

Comment:
@hasufell not sure what has to be done here to take in account LOCAL_CABAL_PROJECT

Reply:
`--keep LOCAL_CABAL_PROJECT`

Reply:
Also, this will break the ci.sh script which still uses this variable.